src/components/MainMenu.js

```diff
@@ -21,20 +21,27 @@ export default function MainMenu() {
   return (
     <Container
       size={10}
-      style={{ justifyContent: 'flex-start', flexFlow: 'row wrap' }}
+      style={{ flexDirection: 'column', justifyContent: 'center' }}
```

Comment:
Why main menu was changed? Also layout is wrong.  Please do not make changes as part of this PR as I already started addressing

Reply:
That again was not my doing

Reply:
so then why do you include in your PR?

Reply:
Because Jon did some stuff on my branch